### PR TITLE
CELDEV-969 - [Security] May 2021 Upgrade Dependencies

### PR DESCRIPTION
**Pull request by fpichler**
https://synjira.atlassian.net/browse/CELDEV-969

### CODE REVIEW

**Review by ebeutler — DISMISSED** [review]

**Comment by fpichler** [comment]
@msladek wdyt?

File: celements-common-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ *
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<parent>
```
Comment (msladek) [inline celements-common-test/pom.xml]:
```suggestion
  <parent>
```
replace all tabs in file

**Review by msladek — CHANGES_REQUESTED** [review]

File: celements-common-test/pom.xml (hunk 1 above)
Comment (fpichler) [reply celements-common-test/pom.xml]: done.

File: celements-common-test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  <version>5.0-SNAPSHOT</version>
+  <description>Celements Shared Tests</description>
+  <scm>
+    <connection>scm:git:git@github.com:celements/celements-shared-tests.git</connection>
+    <developerConnection>scm:git:git@github.com:celements/celements-shared-tests.git</developerConnection>
+    <url>https://github.com/celements/celements-shared-tests/celements-common-test</url>
+    <tag>celements-shared-tests-1.6</tag>
+  </scm>
+  <dependencies>
+		<!-- Add here all your dependencies -->
```
Comment (msladek) [inline celements-common-test/pom.xml]:
```suggestion
    <!-- Add here all your dependencies -->
```
some more tabs ...

File: celements-common-test/pom.xml [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>celements-shared-tests</artifactId>
+  <version>5.0-SNAPSHOT</version>
+  <description>Celements Shared Tests</description>
+  <scm>
+    <connection>scm:git:git@github.com:celements/celements-shared-tests.git</connection>
+    <developerConnection>scm:git:git@github.com:celements/celements-shared-tests.git</developerConnection>
+    <url>https://github.com/celements/celements-shared-tests/celements-common-test</url>
+    <tag>celements-shared-tests-1.6</tag>
```
Comment (msladek) [inline celements-common-test/pom.xml]:
```suggestion
```

**Review by msladek — CHANGES_REQUESTED** [review]

**Review by msladek — DISMISSED** [review]

**Review by msladek — APPROVED** [review]

**Pull request merged**